### DISCUSSION TITLE
Split jump to documentation and definition for custom symbols (Ctrl+Click)

### DISCUSSION THREAD

**Discussion by dalexeev** [godotengine/godot-proposals#5643] [Editor]
Now Ctrl+Click on some identifier in the Script Editor opens:

1. documentation for built-in symbols;
2. definition (script) for custom symbols.

In 4.0-dev, script documentation feature was introduced. It would be nice to be able to jump quickly to both the documentation and the definition. For example, Ctrl+Click opens the documentation, Ctrl+Shift+Click (or maybe Alt+Click) opens the definition.

**Comment by hsandt** [comment]
Direct click to source would be great! I made a related proposal: https://github.com/godotengine/godot-proposals/issues/9095 which would be a two step Go to source (via the F1 help search).

**Comment by Mickeon** [comment]
I find it pretty difficult to discover script documentation in the first place, so any way to make it more accessible would be nice.
